src/db/migrations/18-add-anotherblock.ts

```diff
@@ -0,0 +1,48 @@
+
+import { Knex } from 'knex';
+
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const ANOTHERBLOCK_PLATFORM: MusicPlatform = {
+  id: 'anotherblock',
+  type: MusicPlatformType['single-track-multiprint-contract'],
```

Comment:
err not sure this fits their contract? looks like the contract is for the whole platform and so this could break as soon as they do a second drop?